### PR TITLE
feat(settings): added Motion variables settings

### PR DESCRIPTION
**Pull request by nucliweb**
# SUI Motion 🎞

## Motivation

We need a Motion guideline, to define the baseline to transitions and animations in SUI Components.

## Proposal

We start with settings the variables to define the **durations** and the **timing functions** tokens.

I propose a verbose naming and when define aliases

## Example

```
.element {
  ...
  transition: transform $duration-slow $timing-function-ease-out,
              background-color $duration-slow,
              box-shadow $duration-slow $duration-fast;
  ...
}
```

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@schibstedspain/sui-theme",
-  "version": "8.61.0",
+  "version": "8.67.0",
```
Comment (rmoralp) [inline package-lock.json]: 👀 6 versions? could you also use new @adv-ui naming for this package??

**Comment by nucliweb** [comment]
## Timing reference

times

👉🏼 https://design-system.pluralsight.com/core/motion

File: src/settings/_motion.scss [hunk 2]
```diff
@@ -0,0 +1,27 @@
+// Settings for the SUI Motion Tokens
+
+// Duration Tokens --------------------
+// Naming & timing reference: Polaris Design System https://polaris-tokens.herokuapp.com/
+$duration-none: 0 !default;
+$duration-fast: 100ms !default;
+$duration-base: 200ms !default;
+$duration-slow: 300ms !default;
+$duration-slower: 400ms !default;
```
Comment (turolopezsanabria) [inline src/settings/_motion.scss]: I'd suggest we simplify to 4 options: none, normal, fast and slow (easy to remember and to pick from)
- 0
- 100
- 300
- 500

File: package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
-  "name": "@schibstedspain/sui-theme",
-  "version": "8.67.0",
+  "name": "@a-ui/sui-theme",
```
Comment (rmoralp) [inline package.json]: typo ☝️

**Pull request merged**